Pascal/Pascal.sublime-syntax

```diff
@@ -526,9 +543,10 @@ contexts:
     - match: ''
       pop: 1
 
-###############################################################################
+###[ VARIABLES ]###############################################################
 
 variables:
+  first_line_language: (?:\bpas(cal)?\b)
```

Comment:
Use case of this variable is pretty clear. It always appears standalone or in the middle of another context. Someithing like `{{first_line_language}}?` seems pretty unlikely.

Thus adding it in a group is not necessary.

```suggestion
  first_line_language: \bpas(?:cal)?\b
```

Reply:
Just habit. 🙂